src/state/selectors/polarAxisSelectors.ts

```diff
@@ -19,14 +19,14 @@ export const implicitAngleAxis: AngleAxisSettings = {
   allowDecimals: false,
   allowDuplicatedCategory: false, // defaultPolarAngleAxisProps.allowDuplicatedCategory has it set to true but the actual axis rendering ignores the prop because reasons,
   dataKey: undefined,
-  domain: defaultPolarAngleAxisProps.domain,
+  domain: undefined,
   id: undefined,
   includeHidden: false,
   name: undefined,
   reversed: defaultPolarAngleAxisProps.reversed,
   scale: defaultPolarAngleAxisProps.scale,
   tick: defaultPolarAngleAxisProps.tick,
-  tickCount: defaultPolarAngleAxisProps.tickCount,
+  tickCount: undefined,
```

Comment:
Why make all of these undefined?

Reply:
They are already undefined in the `defaultPolarAngleAxisProps` object, typescript doesn't like that with the strict settings